src/main/java/uk/gov/pay/api/common/SearchConstants.java

```diff
@@ -6,6 +6,7 @@ public class SearchConstants {
     public static final String REFERENCE_KEY = "reference";
     public static final String EMAIL_KEY = "email";
     public static final String STATE_KEY = "state";
+    public static final String STATUS_KEY = "state";
```

Comment:
I think this is the agreement property name

```suggestion
    public static final String STATUS_KEY = "status";
```

Reply:
Updating the test (as described in the previous comment) exposed this oversight and it’s been fixed.